tus_storagehandler/app.py

```diff
@@ -0,0 +1,72 @@
+"""API server entry point."""
+
+import logging
+import os
+from pathlib import Path
+from typing import Optional
+
+from connexion import FlaskApp
+from foca import Foca
+
+logger = logging.getLogger(__name__)
+
+
+def init_app() -> FlaskApp:
+    """Initialize and return the FOCA app.
+
+    This function initializes the FOCA app by loading the configuration
+    from the environment variable `TUS_FOCA_CONFIG_PATH` if set, or from
+    the default path if not. It raises a `FileNotFoundError` if the
+    configuration file is not found.
+
+    Returns:
+        A Connexion application instance.
+
+    Raises:
+        FileNotFoundError: If the configuration file is not found.
+    """
+    # Determine the configuration path
+    if config_path_env := os.getenv("TUS_FOCA_CONFIG_PATH"):
```

Comment:
Make sure you rename this to `CSH_FOCA_CONFIG_PATH` when you rename from `tus_storage_handler` to `cloud_storage_handler` or `csh` (see other comment)